pom.xml

```diff
@@ -763,6 +766,28 @@
                     </execution>
                 </executions>
             </plugin>
+
+            <plugin>
```

Comment:
how it works step 1: maven-deps.txt files are created in module target directories